heltour/tournament/models.py

```diff
@@ -16,6 +16,7 @@
 from django_comments.models import Comment
 from heltour import settings
 import reversion
+from _datetime import date
```

Comment:
Typically it's not a good idea to depend directly on the underlying C libs, which usually start with an underscore. And it's best to depend on the main python one, which does not start with an underscore. 

Is there a reason why you are using the underscore version?

Reply:
No, that was either an auto-import or I saw this in another heltour file. I will change that.